### ISSUE TITLE
[cuptag] - Update Prefix in cuptag.js for Compatibility with tag-import

### ISSUE THREAD

**Issue by difloh** [feederbox826/plugins#16]
In the cuptag.js file, specifically at line 86, the current prefix being used is "[Cup]- " . This prefix creates compatibility issues when the cuptag plugin is used alongside the tag-import plugin.

To ensure both plugins work seamlessly together, I propose changing the prefix from "[Cup]-"  to "[Cup]: " . This modification will help avoid any potential conflicts and improve the overall functionality of both plugins.

Old Code:
```
// constants
const PREFIX = "[Cup]: "
```

New Code:
```
// constants
const PREFIX = "[Cup]- "
```

**Comment by feederbox826** [comment]
Yes that is intentional, tag-import will only import tags that exist and are (hopefully) synced to stashdb. The ignore flag is for my local tags for tracking or non-stashdb tags

**Comment by difloh** [comment]
Thanks, I edited the JavaScript file accordingly to be able to download the images for the cup size tags.

**Comment by feederbox826** [comment]
I'll add a toggle for anyone who wants to match any tag names (ignoring ignore tags)
